.travis.yml

```diff
@@ -4,12 +4,11 @@ jdk:
 - openjdk7
```

Comment:
How can i ran ElasticSearch v5 on travis? I can't get the right java version anywhere it seems.